build.sbt

```diff
@@ -59,28 +60,36 @@ val releaseSettings = Seq(
     commitNextVersion,
     releaseStepCommandAndRemaining("+sonatypeReleaseAll"),
     pushChanges,
-    releaseStepCommand("++2.12.8 docs/publishMicrosite")
+    releaseStepCommand("docs/publishMicrosite")
   )
 )
 
 // scalacOptions += "-Ypartial-unification"
 lazy val coreSettings = Seq(
-  crossScalaVersions := scalaVersion.value :: "2.12.8" :: "2.11.11" :: Nil,
+  crossScalaVersions := scalaVersion.value :: "2.12.11" :: "2.11.12" :: Nil,
   libraryDependencies ++= Seq(
     scalaOrganization.value % "scala-reflect" % scalaVersion.value % "provided",
-    "com.chuusai" %% "shapeless" % "2.3.3",
-    "eu.timepit" %% "singleton-ops" % "0.5.0",
-    "org.typelevel" %% "spire" % "0.17.0-M1",
-    "org.typelevel" %% "spire-laws" % "0.17.0-M1" % "test",
-    "org.scalatest" %% "scalatest" % "3.1.1" % "test"
+    "com.chuusai" %%% "shapeless" % "2.3.3",
+    "eu.timepit" %%% "singleton-ops" % "0.4.3",
```

Comment:
I guess the reason for this downgrade is that `0.5.0` supports scalajs `1.0`, but not `0.6`.
However, we can't move straight to `1.0` as spire doesn't yet support it, and it would be wise to cut a release for `0.6` in any case.

The last version of libra was released on `0.4.0`, so I'm happy making this downgrade, cutting a release, and upgrading as soon as possible.

Reply:
Yes, that's exactly why